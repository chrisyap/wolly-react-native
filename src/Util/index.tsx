import { Dimensions, Platform } from 'react-native';
import DeviceInfo from 'react-native-device-info';

const isAndroid = Platform.OS === 'android';
const isIos = Platform.OS === 'ios';
const isSmallScreen = Dimensions.get('window')?.width <= 360;
const getTestIdProp = (id?: string): object | null =>
  id ? (isIos ? { testID: id } : { accessibilityLabel: id }) : null;
const isTablet = DeviceInfo.isTablet();

const convertToPercent = (value: number): string => {
  return `${value * 100}%`;
};

export { convertToPercent, getTestIdProp, isAndroid, isIos, isSmallScreen, isTablet };
