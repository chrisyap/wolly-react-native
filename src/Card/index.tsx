import React from 'react';
import { ThemeContext } from 'styled-components';
import V, { VProps } from '../View';
import { ThemeProps } from '../Theme';
import { isTablet } from '../Util';

export type CardProps = VProps;

const Card: React.FC<VProps> = (props) => {
  const theme = React.useContext(ThemeContext) as ThemeProps;
  return (
    <V
      bg="white"
      borderTopWidth={isTablet ? 0 : 1}
      borderLeftWidth={isTablet ? 0 : 1}
      borderRightWidth={isTablet ? 0 : 1}
      borderBottomWidth={isTablet ? 0 : 2}
      borderRadius={isTablet ? theme?.radii[3] : theme?.radii[1]}
      borderColor={'secondaries.2'}
      overflow="hidden"
      p={'1px'}
      {...props}
    />
  );
};

export default Card;
