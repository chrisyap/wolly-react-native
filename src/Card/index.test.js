import React from 'react';
import Card from './index';
import Txt from '../Text';
import * as util from '../Util';

jest.mock('../Util');

const isTablet = (utility = util, boolean) => {
  utility.isTablet = boolean;
};

describe('Card', () => {
  beforeEach(() => {
    isTablet(util, false);
  });
  const defaultProps = {
    p: 2,
    children: <Txt>Test</Txt>,
  };
  it('renders correctly', () => {
    const wrapper = shallow(<Card {...defaultProps} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders correctly in tablet', () => {
    isTablet(util, true);
    const wrapper = shallow(<Card {...defaultProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
