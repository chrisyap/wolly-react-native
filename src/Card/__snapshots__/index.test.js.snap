// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<Styled(View)
  bg="white"
  borderBottomWidth={2}
  borderColor="secondaries.2"
  borderLeftWidth={1}
  borderRightWidth={1}
  borderTopWidth={1}
  overflow="hidden"
  p={2}
>
  <Styled(Text)
    allowFontScaling={false}
    color="text"
    fontSize={7}
    theme={
      Object {
        "colors": Object {
          "black": "#000000",
          "blacks": Array [
            "#e7e7e7",
            "#d0d0d0",
            "#b9b9b9",
            "#a2a2a2",
            "#8b8b8b",
            "#737373",
            "#5c5c5c",
            "#454545",
            "#2e2e2e",
            "#171717",
            "#000000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
          ],
          "blueberries": Array [
            "#efebf3",
            "#e0d7e8",
            "#d1c3dc",
            "#c1afd1",
            "#b29bc5",
            "#a387ba",
            "#9473ae",
            "#845fa3",
            "#754b97",
            "#66378c",
            "#572381",
            "#4f1f75",
            "#471c69",
            "#3f195d",
            "#371652",
            "#2f1346",
            "#270f3a",
            "#1f0c2e",
            "#170923",
            "#0f0617",
            "#07030b",
          ],
          "blueberry": "#572381",
          "bluegrey": "#4c626c",
          "bluegreys": Array [
            "#eef0f1",
            "#dee2e4",
            "#ced4d6",
            "#bdc5c9",
            "#adb7bc",
            "#9da9ae",
            "#8d9ba1",
            "#7c8c94",
            "#6c7e86",
            "#5c7079",
            "#4c626c",
            "#455962",
            "#3e5058",
            "#37474e",
            "#303e44",
            "#29353a",
            "#222c31",
            "#1b2327",
            "#141a1d",
            "#0d1113",
            "#060809",
          ],
          "danger": "#d20000",
          "dangerText": "#ffffff",
          "dangers": Array [
            "#fae7e7",
            "#f6d0d0",
            "#f2b9b9",
            "#eea2a2",
            "#ea8b8b",
            "#e67373",
            "#e25c5c",
            "#de4545",
            "#da2e2e",
            "#d61717",
            "#d20000",
            "#be0000",
            "#ab0000",
            "#980000",
            "#850000",
            "#720000",
            "#5f0000",
            "#4c0000",
            "#390000",
            "#260000",
            "#130000",
          ],
          "info": "#005aa3",
          "infoText": "#ffffff",
          "infos": Array [
            "#e7f0f6",
            "#d0e1ee",
            "#b9d2e5",
            "#a2c3dd",
            "#8bb4d5",
            "#73a5cc",
            "#5c96c4",
            "#4587bc",
            "#2e78b3",
            "#1769ab",
            "#005aa3",
            "#005194",
            "#004985",
            "#004176",
            "#003967",
            "#003158",
            "#00284a",
            "#00203b",
            "#00182c",
            "#00101d",
            "#00080e",
          ],
          "lime": "#a4b123",
          "limes": Array [
            "#f6f7eb",
            "#eef0d7",
            "#e6e9c3",
            "#dde2af",
            "#d5db9b",
            "#cdd487",
            "#c5cd73",
            "#bcc65f",
            "#b4bf4b",
            "#acb837",
            "#a4b123",
            "#95a01f",
            "#86901c",
            "#778019",
            "#687016",
            "#596013",
            "#4a500f",
            "#3b400c",
            "#2c3009",
            "#1d2006",
            "#0e1003",
          ],
          "primaries": Array [
            "#f9e7e7",
            "#f3d0d0",
            "#eeb9b9",
            "#e8a2a2",
            "#e38b8b",
            "#dd7373",
            "#d85c5c",
            "#d24545",
            "#cd2e2e",
            "#c71717",
            "#c20000",
            "#b00000",
            "#9e0000",
            "#8d0000",
            "#7b0000",
            "#690000",
            "#580000",
            "#460000",
            "#340000",
            "#230000",
            "#100",
          ],
          "primary": "#c20000",
          "primaryText": "#ffffff",
          "red": "#c20000",
          "reds": Array [
            "#f9e7e7",
            "#f3d0d0",
            "#eeb9b9",
            "#e8a2a2",
            "#e38b8b",
            "#dd7373",
            "#d85c5c",
            "#d24545",
            "#cd2e2e",
            "#c71717",
            "#c20000",
            "#b00000",
            "#9e0000",
            "#8d0000",
            "#7b0000",
            "#690000",
            "#580000",
            "#460000",
            "#340000",
            "#230000",
            "#100",
          ],
          "sea": "#152773",
          "seas": Array [
            "#e9ebf2",
            "#d4d7e5",
            "#bfc4d8",
            "#a9b0cc",
            "#949cbf",
            "#7f89b2",
            "#6a75a5",
            "#546199",
            "#3f4e8c",
            "#2a3a7f",
            "#152773",
            "#132368",
            "#111f5e",
            "#0f1c53",
            "#0d1849",
            "#0b153e",
            "#091134",
            "#070e29",
            "#050a1f",
            "#030714",
            "#01030a",
          ],
          "secondaries": Array [
            "#eef0f1",
            "#dee2e4",
            "#ced4d6",
            "#bdc5c9",
            "#adb7bc",
            "#9da9ae",
            "#8d9ba1",
            "#7c8c94",
            "#6c7e86",
            "#5c7079",
            "#4c626c",
            "#455962",
            "#3e5058",
            "#37474e",
            "#303e44",
            "#29353a",
            "#222c31",
            "#1b2327",
            "#141a1d",
            "#0d1113",
            "#060809",
          ],
          "secondary": "#4c626c",
          "secondaryAccent": "#4c626c",
          "secondaryAccentText": "#ffffff",
          "skies": Array [
            "#e7f0f6",
            "#d0e1ee",
            "#b9d2e5",
            "#a2c3dd",
            "#8bb4d5",
            "#73a5cc",
            "#5c96c4",
            "#4587bc",
            "#2e78b3",
            "#1769ab",
            "#005aa3",
            "#005194",
            "#004985",
            "#004176",
            "#003967",
            "#003158",
            "#00284a",
            "#00203b",
            "#00182c",
            "#00101d",
            "#00080e",
          ],
          "sky": "#005aa3",
          "success": "#037d04",
          "successText": "#ffffff",
          "successes": Array [
            "#e8f3e8",
            "#d1e7d1",
            "#badbba",
            "#a3cfa3",
            "#8cc38c",
            "#75b876",
            "#5eac5f",
            "#47a048",
            "#309431",
            "#19881a",
            "#037d04",
            "#027103",
            "#026603",
            "#025a02",
            "#014f02",
            "#014402",
            "#013801",
            "#012d01",
            "#002201",
            "#001600",
            "#000b00",
          ],
          "tangerine": "#e9600e",
          "tangerines": Array [
            "#fdf0e9",
            "#fbe2d3",
            "#f9d3bd",
            "#f7c5a7",
            "#f5b691",
            "#f3a87b",
            "#f19965",
            "#ef8b4f",
            "#ed7c39",
            "#eb6e23",
            "#e9600e",
            "#d3570c",
            "#be4e0b",
            "#a9450a",
            "#943d08",
            "#7f3407",
            "#692b06",
            "#542205",
            "#3f1a03",
            "#2a1102",
            "#150801",
          ],
          "text": "#111111",
          "warning": "#F36E20",
          "warningText": "#000000",
          "warnings": Array [
            "#fdf1ea",
            "#fce4d6",
            "#fbd7c2",
            "#facaad",
            "#f9bd99",
            "#f8af85",
            "#f7a271",
            "#f6955c",
            "#f58848",
            "#f47b34",
            "#F36E20",
            "#dc641d",
            "#c65a1a",
            "#b05017",
            "#9a4614",
            "#843b11",
            "#6e320e",
            "#58280b",
            "#421d08",
            "#2c1305",
            "#160902",
          ],
          "white": "#ffffff",
        },
        "duration": Object {
          "fast": "150ms",
          "normal": "300ms",
          "slow": "450ms",
          "slowest": "600ms",
        },
        "fontSizes": Array [
          "8px",
          "9px",
          "10px",
          "11px",
          "12px",
          "13px",
          "14px",
          "16px",
          "18px",
          "20px",
          "24px",
          "26px",
          "28px",
          "32px",
          "34px",
          "41px",
          "46px",
          "52px",
          "58px",
          "66px",
          "74px",
          "83px",
          "94px",
          "105px",
        ],
        "fontWeights": Object {
          "bold": 700,
          "light": 300,
          "medium": 500,
          "normal": 400,
          "semibold": 600,
        },
        "lineHeights": Object {
          "display": "20px",
          "standard": "22px",
        },
        "radii": Array [
          "0px",
          "2px",
          "4px",
          "8px",
          "16px",
        ],
        "radius": "2px",
        "radiusRounded": "290486px",
        "shadows": Array [
          "
  0 0px 0px 0 rgba(76,98,108,0.1), 
  0 0px 0px 0 rgba(76,98,108,0.1)",
          "
  0 0.75px 1.25px 0 rgba(76,98,108,0.1), 
  0 0.5px 2.5px 0 rgba(76,98,108,0.1)",
          "
  0 1.5px 2.5px 0 rgba(76,98,108,0.1), 
  0 1px 5px 0 rgba(76,98,108,0.1)",
          "
  0 2.25px 3.75px 0 rgba(76,98,108,0.1), 
  0 1.5px 7.5px 0 rgba(76,98,108,0.1)",
          "
  0 3px 5px 0 rgba(76,98,108,0.1), 
  0 2px 10px 0 rgba(76,98,108,0.1)",
          "
  0 3.75px 6.25px 0 rgba(76,98,108,0.1), 
  0 2.5px 12.5px 0 rgba(76,98,108,0.1)",
          "
  0 4.5px 7.5px 0 rgba(76,98,108,0.1), 
  0 3px 15px 0 rgba(76,98,108,0.1)",
          "
  0 5.25px 8.75px 0 rgba(76,98,108,0.1), 
  0 3.5px 17.5px 0 rgba(76,98,108,0.1)",
          "
  0 6px 10px 0 rgba(76,98,108,0.1), 
  0 4px 20px 0 rgba(76,98,108,0.1)",
          "
  0 6.75px 11.25px 0 rgba(76,98,108,0.1), 
  0 4.5px 22.5px 0 rgba(76,98,108,0.1)",
          "
  0 7.5px 12.5px 0 rgba(76,98,108,0.1), 
  0 5px 25px 0 rgba(76,98,108,0.1)",
          "
  0 8.25px 13.75px 0 rgba(76,98,108,0.1), 
  0 5.5px 27.5px 0 rgba(76,98,108,0.1)",
          "
  0 9px 15px 0 rgba(76,98,108,0.1), 
  0 6px 30px 0 rgba(76,98,108,0.1)",
          "
  0 9.75px 16.25px 0 rgba(76,98,108,0.1), 
  0 6.5px 32.5px 0 rgba(76,98,108,0.1)",
          "
  0 10.5px 17.5px 0 rgba(76,98,108,0.1), 
  0 7px 35px 0 rgba(76,98,108,0.1)",
          "
  0 11.25px 18.75px 0 rgba(76,98,108,0.1), 
  0 7.5px 37.5px 0 rgba(76,98,108,0.1)",
          "
  0 12px 20px 0 rgba(76,98,108,0.1), 
  0 8px 40px 0 rgba(76,98,108,0.1)",
          "
  0 12.75px 21.25px 0 rgba(76,98,108,0.1), 
  0 8.5px 42.5px 0 rgba(76,98,108,0.1)",
          "
  0 13.5px 22.5px 0 rgba(76,98,108,0.1), 
  0 9px 45px 0 rgba(76,98,108,0.1)",
          "
  0 14.25px 23.75px 0 rgba(76,98,108,0.1), 
  0 9.5px 47.5px 0 rgba(76,98,108,0.1)",
          "
  0 15px 25px 0 rgba(76,98,108,0.1), 
  0 10px 50px 0 rgba(76,98,108,0.1)",
          "
  0 15.75px 26.25px 0 rgba(76,98,108,0.1), 
  0 10.5px 52.5px 0 rgba(76,98,108,0.1)",
          "
  0 16.5px 27.5px 0 rgba(76,98,108,0.1), 
  0 11px 55px 0 rgba(76,98,108,0.1)",
          "
  0 17.25px 28.75px 0 rgba(76,98,108,0.1), 
  0 11.5px 57.5px 0 rgba(76,98,108,0.1)",
          "
  0 18px 30px 0 rgba(76,98,108,0.1), 
  0 12px 60px 0 rgba(76,98,108,0.1)",
        ],
        "space": Array [
          "0px",
          "8px",
          "16px",
          "24px",
          "32px",
          "40px",
          "48px",
          "56px",
          "64px",
          "72px",
          "80px",
          "88px",
          "96px",
          "104px",
          "112px",
          "120px",
          "128px",
          "136px",
          "144px",
          "152px",
        ],
        "spacer": 16,
        "timingFunctions": Object {
          "easeIn": "cubic-bezier(0.5, 0, 1, 1)",
          "easeInOut": "cubic-bezier(0.5, 0, 0.25, 1)",
          "easeOut": "cubic-bezier(0, 0, 0.25, 1)",
        },
        "transitionDelays": Object {
          "large": "260ms",
          "medium": "160ms",
          "small": "60ms",
          "xLarge": "360ms",
        },
      }
    }
  >
    Test
  </Styled(Text)>
</Styled(View)>
`;

exports[`Card renders correctly in tablet 1`] = `
<Styled(View)
  bg="white"
  borderBottomWidth={0}
  borderColor="secondaries.2"
  borderLeftWidth={0}
  borderRightWidth={0}
  borderTopWidth={0}
  overflow="hidden"
  p={2}
>
  <Styled(Text)
    allowFontScaling={false}
    color="text"
    fontSize={7}
    theme={
      Object {
        "colors": Object {
          "black": "#000000",
          "blacks": Array [
            "#e7e7e7",
            "#d0d0d0",
            "#b9b9b9",
            "#a2a2a2",
            "#8b8b8b",
            "#737373",
            "#5c5c5c",
            "#454545",
            "#2e2e2e",
            "#171717",
            "#000000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
            "#000",
          ],
          "blueberries": Array [
            "#efebf3",
            "#e0d7e8",
            "#d1c3dc",
            "#c1afd1",
            "#b29bc5",
            "#a387ba",
            "#9473ae",
            "#845fa3",
            "#754b97",
            "#66378c",
            "#572381",
            "#4f1f75",
            "#471c69",
            "#3f195d",
            "#371652",
            "#2f1346",
            "#270f3a",
            "#1f0c2e",
            "#170923",
            "#0f0617",
            "#07030b",
          ],
          "blueberry": "#572381",
          "bluegrey": "#4c626c",
          "bluegreys": Array [
            "#eef0f1",
            "#dee2e4",
            "#ced4d6",
            "#bdc5c9",
            "#adb7bc",
            "#9da9ae",
            "#8d9ba1",
            "#7c8c94",
            "#6c7e86",
            "#5c7079",
            "#4c626c",
            "#455962",
            "#3e5058",
            "#37474e",
            "#303e44",
            "#29353a",
            "#222c31",
            "#1b2327",
            "#141a1d",
            "#0d1113",
            "#060809",
          ],
          "danger": "#d20000",
          "dangerText": "#ffffff",
          "dangers": Array [
            "#fae7e7",
            "#f6d0d0",
            "#f2b9b9",
            "#eea2a2",
            "#ea8b8b",
            "#e67373",
            "#e25c5c",
            "#de4545",
            "#da2e2e",
            "#d61717",
            "#d20000",
            "#be0000",
            "#ab0000",
            "#980000",
            "#850000",
            "#720000",
            "#5f0000",
            "#4c0000",
            "#390000",
            "#260000",
            "#130000",
          ],
          "info": "#005aa3",
          "infoText": "#ffffff",
          "infos": Array [
            "#e7f0f6",
            "#d0e1ee",
            "#b9d2e5",
            "#a2c3dd",
            "#8bb4d5",
            "#73a5cc",
            "#5c96c4",
            "#4587bc",
            "#2e78b3",
            "#1769ab",
            "#005aa3",
            "#005194",
            "#004985",
            "#004176",
            "#003967",
            "#003158",
            "#00284a",
            "#00203b",
            "#00182c",
            "#00101d",
            "#00080e",
          ],
          "lime": "#a4b123",
          "limes": Array [
            "#f6f7eb",
            "#eef0d7",
            "#e6e9c3",
            "#dde2af",
            "#d5db9b",
            "#cdd487",
            "#c5cd73",
            "#bcc65f",
            "#b4bf4b",
            "#acb837",
            "#a4b123",
            "#95a01f",
            "#86901c",
            "#778019",
            "#687016",
            "#596013",
            "#4a500f",
            "#3b400c",
            "#2c3009",
            "#1d2006",
            "#0e1003",
          ],
          "primaries": Array [
            "#f9e7e7",
            "#f3d0d0",
            "#eeb9b9",
            "#e8a2a2",
            "#e38b8b",
            "#dd7373",
            "#d85c5c",
            "#d24545",
            "#cd2e2e",
            "#c71717",
            "#c20000",
            "#b00000",
            "#9e0000",
            "#8d0000",
            "#7b0000",
            "#690000",
            "#580000",
            "#460000",
            "#340000",
            "#230000",
            "#100",
          ],
          "primary": "#c20000",
          "primaryText": "#ffffff",
          "red": "#c20000",
          "reds": Array [
            "#f9e7e7",
            "#f3d0d0",
            "#eeb9b9",
            "#e8a2a2",
            "#e38b8b",
            "#dd7373",
            "#d85c5c",
            "#d24545",
            "#cd2e2e",
            "#c71717",
            "#c20000",
            "#b00000",
            "#9e0000",
            "#8d0000",
            "#7b0000",
            "#690000",
            "#580000",
            "#460000",
            "#340000",
            "#230000",
            "#100",
          ],
          "sea": "#152773",
          "seas": Array [
            "#e9ebf2",
            "#d4d7e5",
            "#bfc4d8",
            "#a9b0cc",
            "#949cbf",
            "#7f89b2",
            "#6a75a5",
            "#546199",
            "#3f4e8c",
            "#2a3a7f",
            "#152773",
            "#132368",
            "#111f5e",
            "#0f1c53",
            "#0d1849",
            "#0b153e",
            "#091134",
            "#070e29",
            "#050a1f",
            "#030714",
            "#01030a",
          ],
          "secondaries": Array [
            "#eef0f1",
            "#dee2e4",
            "#ced4d6",
            "#bdc5c9",
            "#adb7bc",
            "#9da9ae",
            "#8d9ba1",
            "#7c8c94",
            "#6c7e86",
            "#5c7079",
            "#4c626c",
            "#455962",
            "#3e5058",
            "#37474e",
            "#303e44",
            "#29353a",
            "#222c31",
            "#1b2327",
            "#141a1d",
            "#0d1113",
            "#060809",
          ],
          "secondary": "#4c626c",
          "secondaryAccent": "#4c626c",
          "secondaryAccentText": "#ffffff",
          "skies": Array [
            "#e7f0f6",
            "#d0e1ee",
            "#b9d2e5",
            "#a2c3dd",
            "#8bb4d5",
            "#73a5cc",
            "#5c96c4",
            "#4587bc",
            "#2e78b3",
            "#1769ab",
            "#005aa3",
            "#005194",
            "#004985",
            "#004176",
            "#003967",
            "#003158",
            "#00284a",
            "#00203b",
            "#00182c",
            "#00101d",
            "#00080e",
          ],
          "sky": "#005aa3",
          "success": "#037d04",
          "successText": "#ffffff",
          "successes": Array [
            "#e8f3e8",
            "#d1e7d1",
            "#badbba",
            "#a3cfa3",
            "#8cc38c",
            "#75b876",
            "#5eac5f",
            "#47a048",
            "#309431",
            "#19881a",
            "#037d04",
            "#027103",
            "#026603",
            "#025a02",
            "#014f02",
            "#014402",
            "#013801",
            "#012d01",
            "#002201",
            "#001600",
            "#000b00",
          ],
          "tangerine": "#e9600e",
          "tangerines": Array [
            "#fdf0e9",
            "#fbe2d3",
            "#f9d3bd",
            "#f7c5a7",
            "#f5b691",
            "#f3a87b",
            "#f19965",
            "#ef8b4f",
            "#ed7c39",
            "#eb6e23",
            "#e9600e",
            "#d3570c",
            "#be4e0b",
            "#a9450a",
            "#943d08",
            "#7f3407",
            "#692b06",
            "#542205",
            "#3f1a03",
            "#2a1102",
            "#150801",
          ],
          "text": "#111111",
          "warning": "#F36E20",
          "warningText": "#000000",
          "warnings": Array [
            "#fdf1ea",
            "#fce4d6",
            "#fbd7c2",
            "#facaad",
            "#f9bd99",
            "#f8af85",
            "#f7a271",
            "#f6955c",
            "#f58848",
            "#f47b34",
            "#F36E20",
            "#dc641d",
            "#c65a1a",
            "#b05017",
            "#9a4614",
            "#843b11",
            "#6e320e",
            "#58280b",
            "#421d08",
            "#2c1305",
            "#160902",
          ],
          "white": "#ffffff",
        },
        "duration": Object {
          "fast": "150ms",
          "normal": "300ms",
          "slow": "450ms",
          "slowest": "600ms",
        },
        "fontSizes": Array [
          "8px",
          "9px",
          "10px",
          "11px",
          "12px",
          "13px",
          "14px",
          "16px",
          "18px",
          "20px",
          "24px",
          "26px",
          "28px",
          "32px",
          "34px",
          "41px",
          "46px",
          "52px",
          "58px",
          "66px",
          "74px",
          "83px",
          "94px",
          "105px",
        ],
        "fontWeights": Object {
          "bold": 700,
          "light": 300,
          "medium": 500,
          "normal": 400,
          "semibold": 600,
        },
        "lineHeights": Object {
          "display": "20px",
          "standard": "22px",
        },
        "radii": Array [
          "0px",
          "2px",
          "4px",
          "8px",
          "16px",
        ],
        "radius": "2px",
        "radiusRounded": "290486px",
        "shadows": Array [
          "
  0 0px 0px 0 rgba(76,98,108,0.1), 
  0 0px 0px 0 rgba(76,98,108,0.1)",
          "
  0 0.75px 1.25px 0 rgba(76,98,108,0.1), 
  0 0.5px 2.5px 0 rgba(76,98,108,0.1)",
          "
  0 1.5px 2.5px 0 rgba(76,98,108,0.1), 
  0 1px 5px 0 rgba(76,98,108,0.1)",
          "
  0 2.25px 3.75px 0 rgba(76,98,108,0.1), 
  0 1.5px 7.5px 0 rgba(76,98,108,0.1)",
          "
  0 3px 5px 0 rgba(76,98,108,0.1), 
  0 2px 10px 0 rgba(76,98,108,0.1)",
          "
  0 3.75px 6.25px 0 rgba(76,98,108,0.1), 
  0 2.5px 12.5px 0 rgba(76,98,108,0.1)",
          "
  0 4.5px 7.5px 0 rgba(76,98,108,0.1), 
  0 3px 15px 0 rgba(76,98,108,0.1)",
          "
  0 5.25px 8.75px 0 rgba(76,98,108,0.1), 
  0 3.5px 17.5px 0 rgba(76,98,108,0.1)",
          "
  0 6px 10px 0 rgba(76,98,108,0.1), 
  0 4px 20px 0 rgba(76,98,108,0.1)",
          "
  0 6.75px 11.25px 0 rgba(76,98,108,0.1), 
  0 4.5px 22.5px 0 rgba(76,98,108,0.1)",
          "
  0 7.5px 12.5px 0 rgba(76,98,108,0.1), 
  0 5px 25px 0 rgba(76,98,108,0.1)",
          "
  0 8.25px 13.75px 0 rgba(76,98,108,0.1), 
  0 5.5px 27.5px 0 rgba(76,98,108,0.1)",
          "
  0 9px 15px 0 rgba(76,98,108,0.1), 
  0 6px 30px 0 rgba(76,98,108,0.1)",
          "
  0 9.75px 16.25px 0 rgba(76,98,108,0.1), 
  0 6.5px 32.5px 0 rgba(76,98,108,0.1)",
          "
  0 10.5px 17.5px 0 rgba(76,98,108,0.1), 
  0 7px 35px 0 rgba(76,98,108,0.1)",
          "
  0 11.25px 18.75px 0 rgba(76,98,108,0.1), 
  0 7.5px 37.5px 0 rgba(76,98,108,0.1)",
          "
  0 12px 20px 0 rgba(76,98,108,0.1), 
  0 8px 40px 0 rgba(76,98,108,0.1)",
          "
  0 12.75px 21.25px 0 rgba(76,98,108,0.1), 
  0 8.5px 42.5px 0 rgba(76,98,108,0.1)",
          "
  0 13.5px 22.5px 0 rgba(76,98,108,0.1), 
  0 9px 45px 0 rgba(76,98,108,0.1)",
          "
  0 14.25px 23.75px 0 rgba(76,98,108,0.1), 
  0 9.5px 47.5px 0 rgba(76,98,108,0.1)",
          "
  0 15px 25px 0 rgba(76,98,108,0.1), 
  0 10px 50px 0 rgba(76,98,108,0.1)",
          "
  0 15.75px 26.25px 0 rgba(76,98,108,0.1), 
  0 10.5px 52.5px 0 rgba(76,98,108,0.1)",
          "
  0 16.5px 27.5px 0 rgba(76,98,108,0.1), 
  0 11px 55px 0 rgba(76,98,108,0.1)",
          "
  0 17.25px 28.75px 0 rgba(76,98,108,0.1), 
  0 11.5px 57.5px 0 rgba(76,98,108,0.1)",
          "
  0 18px 30px 0 rgba(76,98,108,0.1), 
  0 12px 60px 0 rgba(76,98,108,0.1)",
        ],
        "space": Array [
          "0px",
          "8px",
          "16px",
          "24px",
          "32px",
          "40px",
          "48px",
          "56px",
          "64px",
          "72px",
          "80px",
          "88px",
          "96px",
          "104px",
          "112px",
          "120px",
          "128px",
          "136px",
          "144px",
          "152px",
        ],
        "spacer": 16,
        "timingFunctions": Object {
          "easeIn": "cubic-bezier(0.5, 0, 1, 1)",
          "easeInOut": "cubic-bezier(0.5, 0, 0.25, 1)",
          "easeOut": "cubic-bezier(0, 0, 0.25, 1)",
        },
        "transitionDelays": Object {
          "large": "260ms",
          "medium": "160ms",
          "small": "60ms",
          "xLarge": "360ms",
        },
      }
    }
  >
    Test
  </Styled(Text)>
</Styled(View)>
`;
